### ISSUE TITLE
Rule Change: option to not require semicolons after class properties

### ISSUE THREAD

**Issue by lorenzos** [eslint/eslint#15381]
### What rule do you want to change?

`semi`.

### What change to do you want to make?

Generate fewer warnings.

### How do you think the change should be implemented?

A new option.

### Example code

```js
export default class MyClass {

	static a = 0 // <--
	b = 1 // <--
	
	foo() {
		return 2;
	}
	
	bar = () => {
		return 3;
	} // <--

}
```

### What does the rule currently do for this code?

In the example above, `semi` currently requires semicolons after the declarations of `a`, `b` and `bar` (class properties), but not for `foo` (method).

### What will the rule do after it's changed?

It'd be great to have an option to exclude semicolons checks for class properties, but still requires them everywhere else.

### Participation

- [ ] I am willing to submit a pull request to implement this change.

### Additional comments

This was actually the previous behavior, before `semi` was updated to support class properties, I think in #14857.

**Comment by nzakas** [comment]
Thanks for the input. As [stated in the README](https://github.com/eslint/eslint#stylistic-rule-updates), we actually aren't making any further updates to rules that enforce stylistic preferences. If you have a strong preference for such a change, you can always copy the existing rule into a custom rule and make it behave the way you want.
